### PR TITLE
make compiler diagnostic analyzer to always run first in IDE.

### PR DESCRIPTION
**Pull request by heejaechang**

### CODE REVIEW

**Comment by heejaechang** [comment]
@mavasani @srivatsn @shyamnamboodiripad @tmeschter can you take a look? thank you

File: src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.StateManager.HostStates.cs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using Roslyn.Utilities;
```
Comment (mavasani) [inline src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.StateManager.HostStates.cs]: Sort usings?

**Comment by mavasani** [comment]
:+1: Thank you for fixing this!

**Comment by tmeschter** [comment]
LGTM

**Comment by heejaechang** [comment]
@dotnet-bot retest this, please

**Comment by heejaechang** [comment]
@dotnet-bot retest this, please.

**Comment by heejaechang** [comment]
@dotnet-bot retest this, please.

**Comment by heejaechang** [comment]
@dotnet-bot retest this please

**Pull request merged**